src/gumbo.h

```diff
@@ -36,7 +36,7 @@
  * @endcode
  * HTML5 Spec:
  *
- * http://www.whatwg.org/specs/web-apps/current-work/multipage/syntax.html
+ * http://www.whatwg.org/specs/web-apps/ui-work/multipage/syntax.html
```

Comment:
Curious about the comment changes in here and other source files.  Have the spec files been moved?  For me, the new links 404, while the old ones still work.  Or is it a simple find & replace snafu.